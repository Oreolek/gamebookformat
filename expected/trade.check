examples/trade.json: Could not reach section 'someotherplace' from start.
