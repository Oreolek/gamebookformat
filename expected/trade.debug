BEGIN DEBUG OUTPUT
Book title: Gamebook
Number of sections: 2
Introduction
 Testing trade of things. You can buy things using [COUNT gold]Gold[/COUNT] of which you start with [SET gold]100[/SET]. You also have an [COLLECT item]Inventory[/COLLECT]. At start you carry a [ADD item]sword[/ADD]. 


Turn to 1 to begin.
1 (start) -  You can buy a [FOUND item]shield[/FOUND] here for 50 Gold. There is also a [FOUND item]chain mail[/FOUND] costing 150. You can pick up some [FOUND item]spiderweb[/FOUND] for free. You can go on to the end at 2. 
2 (theend) -  It ends here.
END DEBUG OUTPUT
