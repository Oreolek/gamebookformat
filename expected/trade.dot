digraph gamebook {

 shield50chain mail150spiderweb1->2
}
