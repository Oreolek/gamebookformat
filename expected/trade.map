{"start": 1, "other": 4, "theend": 2, "someotherplace": 3}