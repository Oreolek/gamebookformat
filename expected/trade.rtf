{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Gamebook
\b0\
\
\b \qc Introduction
\b0\



\ql  Testing trade of things. You can buy things using Gold of which you start with 100. You also have an Inventory. At start you carry a sword.  \
\
\b Turn to 1 to begin.
\b0\
\
\b \qc 1
\b0\
\ql  You can buy a shield here for 50 Gold or a beautiful gold ring for only 75. There is also a chain mail costing 150. You can pick up some spiderweb for free. You can go to \b 4
\b0
 at a cost of 50. You can go on to the end at \b 2
\b0
.  \
\

\b \qc 2
\b0\
\ql  It ends here. There is a crown here that you can trade for some spiderweb.  \
\

\b \qc 3
\b0\
\ql  This is somewhere else. Congratulations you have found 100 gold! Go back to \b 1
\b0
.  \
\

\b \qc 4
\b0\
\ql  Now you have 50 less gold than you just had. You can go back to \b 1
\b0
.  \
\

}
