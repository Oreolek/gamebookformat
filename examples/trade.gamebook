= Introduction
Testing trade of things. You can buy things using
[count gold]Gold[/count] of which you start with [set gold]100[/set].
You also have an [collect item]Inventory[/item]. At start you carry
a [add item]sword[/add].

* 1 start
You can buy a [found item]shield[/found] here for [cost gold]50[/cost] Gold
or a beautiful [found item]gold ring[/found] for only [cost gold]75[/cost].
There is also a [found item]chain mail[/found] costing [cost gold]150[/cost].
You can pick up some [found item]spiderweb[/found] for free.
You can go to [[other]] at a cost of [cost gold]50[/cost].
You can go on to the end at [[theend]].

* other
Hopefully you now have 50 less gold than you just had.
You can go back to [[start]].

* theend
It ends here.
