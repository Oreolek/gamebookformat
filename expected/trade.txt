    Gamebook

Introduction
 Testing trade of things. You can buy things using Gold of which you start with 100. You also have an Inventory. At start you carry a sword. 


Turn to 1 to begin.
1
 You can buy a shield here for 50 Gold. There is also a chain mail costing 150. You can pick up some spiderweb for free. You can go on to the end at 2. 

2
 It ends here.

